src/jv_parse.c

```diff
@@ -512,6 +512,8 @@ static pfunc check_literal(struct jv_parser* p) {
   switch (p->tokenbuf[0]) {
   case 't': pattern = "true"; plen = 4; v = jv_true(); break;
   case 'f': pattern = "false"; plen = 5; v = jv_false(); break;
+  case '\'':
+    return "Invalid string literal";
```

Comment:
Maybe should even say something like "Invalid string literal. Try use double quote instead of single quote."

Reply:
@wader If you want that, add it to the lexer, or it will also be printed for `fromjson/0` or `tonumber/0` or JSON parsing errors